js/components/discoveryPanel.js

```diff
@@ -221,7 +241,10 @@ DiscoveryPanel.defaultProps = {
     relatedVideos: []
   },
   controller: {
-    sendDiscoveryClickEvent: function(a,b){}
+    sendDiscoveryClickEvent: function(a,b){},
+    sendDiscoveryDisplayEvent: function(a,b,c,d,e){}
```

Comment:
Maybe I'm getting confused but here this looks like a temp/stub function. In the unit test changes below(tests/controller-test.js) you are sending actual values for each param.  Where are those defined?

Reply:
@pilievOoyala I think this stub is needed for unit tests in discoveryPanel-test.js. Because of the changes for analytics.js sendDiscoveryDisplayEvent is now fired from within the UI element and not the controller. In fact, that unit test in controller-test.js was wrong, it should be testing the old function sendDiscoveryDisplayEvent function that was fired in controller.js (renamed sendDiscoveryDisplayEventOld) that we need to keep around in order to still have a path for sending discovery events to reporter.js (until it is removed completely). I updated the controller-test.js unit test fix that now